README.md

```diff
@@ -1,62 +1,74 @@
+## Building and Running
 
-```
-mbed compile -m K64F -t ARM
-```
+1. Connect a USB cable between the USB port on the target and the host computer.
+2. Run the following command to build the example project and program the microcontroller flash memory:
+    ```bash
+    $ mbed compile -m <TARGET> -t <TOOLCHAIN> --flash --sterm
+    ```
+(Note: Mbed CLI command-line option "--sterm" is used to open a serial terminal after flashing.)
+
+Your PC may take a few minutes to compile your code.
 
-Your PC may take a few minutes to compile your code. At the end, you see the following result:
+The binary is located at `./BUILD/<TARGET>/<TOOLCHAIN>/mbed-os-example-cpu-stats.bin` and can alternatively be manually copied to the target which gets mounted on the host computer via USB.
 
+Depending on the target, the example project can be built with GCC_ARM, ARM or IAR toolchain. Run the command below after installing ARM Mbed CLI to determine which toolchain supports your target.
+
+```bash
+$ mbed compile -S
 ```
-[snip]
-+------------------+-------+-------+------+
-| Module           | .text | .data | .bss |
-+------------------+-------+-------+------+
-| [lib]\c_w.l      | 11473 |    16 |  348 |
-| [lib]\cpprt_w.l  |    36 |     0 |    0 |
-| [lib]\fz_wm.l    |    18 |     0 |    0 |
-| [lib]\m_wm.l     |    48 |     0 |    0 |
-| anon$$obj.o      |    32 |     0 | 1024 |
-| main.o           |   530 |     8 |    0 |
-| mbed-os\drivers  |   165 |     0 |    0 |
-| mbed-os\events   |  2139 |     8 | 1568 |
-| mbed-os\features |   132 |     0 |  304 |
-| mbed-os\hal      |  1910 |    34 |  128 |
-| mbed-os\platform |  3695 |   104 |  604 |
-| mbed-os\rtos     | 17302 |  2310 | 4592 |
-| mbed-os\targets  | 10707 |   112 |  324 |
-| Subtotals        | 48187 |  2592 | 8892 |
-+------------------+-------+-------+------+
-Total Static RAM memory (data + bss): 11484 bytes
-Total Flash memory (text + data): 50779 bytes
-
-Image: .\BUILD\K64F\ARM\mbed-os-example-cpu-stats.bin
 
+## Expected output
+The LED blinks and the serial terminal shows an output similar to the following:
+```
+--- Terminal on /dev/ttyACM0 - 9600,8,N,1 ---
+Uptime:  2000061 Idle Time:  1999817 Sleep time: 0 DeepSleep time:  1999817 Idle: 99% Usage: 1%
+Uptime:  4000061 Idle Time:  3898285 Sleep time: 0 DeepSleep time:  3898285 Idle: 94% Usage: 6%
+Uptime:  6000061 Idle Time:  5797576 Sleep time: 0 DeepSleep time:  5797576 Idle: 94% Usage: 6%
+Uptime:  8000061 Idle Time:  7697295 Sleep time: 0 DeepSleep time:  7697295 Idle: 94% Usage: 6%
+Uptime: 10000061 Idle Time:  9596557 Sleep time: 0 DeepSleep time:  9596557 Idle: 94% Usage: 6%
+Uptime: 12000061 Idle Time: 11496370 Sleep time: 0 DeepSleep time: 11496370 Idle: 94% Usage: 6%
+Uptime: 14000061 Idle Time: 13395907 Sleep time: 0 DeepSleep time: 13395907 Idle: 94% Usage: 6%
+Uptime: 16000030 Idle Time: 14295200 Sleep time: 0 DeepSleep time: 14295200 Idle: 44% Usage: 56%
+```
+
+Below describes information about the above logs:
```

Comment:
Suggestion: `You can find below some information on how to interpret the above fields:`